packages/ocamlfind/ocamlfind.1.5.6/url

```diff
@@ -1,2 +1,3 @@
 archive: "http://download.camlcity.org/download/findlib-1.5.6.tar.gz"
 checksum: "91585dd5459cb69bfd9a0689bf222403"
+mirrors: [ "http://download2.camlcity.org/download/findlib-1.5.7.tar.gz" ]
```

Comment:
Mismatch here - `1.5.7` in the mirror, should be probably be `1.5.6`.

Reply:
Indeed. Fixed by a forced push.

Reply:
Thanks!